pkl-core/src/test/files/LanguageSnippetTests/input/packages/badRead8.error.pkl

```diff
@@ -1 +1 @@
-res = import("package://localhost:0/badImportsWithinPackage@1.0.0#/unknownDependencyReaad.pkl")
+res = import("package://localhost:0/badImportsWithinPackage@1.0.0#/unknownDependencyRead.pkl")
```

Comment:
Unlike `Ostritch`, these files don't seem to exist (which more or less makes sense given what they're testing) so there doesn't seem to be something to rename...